### DISCUSSION TITLE
How to automatically trigger some effect when two global store change + meet some condition

### DISCUSSION THREAD

**Discussion by nhattran998** [solidjs/solid#1743] [Q&A]
Hi everyone, I have a problem like this

I have the AuthStore and WorkStore(both nested Object and created from `createRoot`), in case if user's type = newbie and did some work done --> Auto trigger updating the other store/effect. This thing seems like reactive programming, like `Rxjs`.

But in the current context use Solid, do you know how I can do this?

Thank you for your help!

**Comment by atk** [comment]
Just use an createEffect with the conditions. The effect will re-run whenever the store value changes.
